<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>0</y>
     <width>650</width>
     <height>781</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="username">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>360</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>굴림</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>사용자 이름</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_enter">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>450</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QListWidget" name="chatroom">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>510</width>
       <height>581</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>테스트</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>test2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>test3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>test4</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>채팅 목록</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_exit">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>나가기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_makechat">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>90</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>채팅방 만들기</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newchatname">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>95</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btn_send">
     <property name="geometry">
      <rect>
       <x>494</x>
       <y>646</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>전송</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sendmessage">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>645</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_back">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
    <widget class="QListWidget" name="receivemessage">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>80</y>
       <width>521</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
